cpp/log/frontend_signer_test.cc

```diff
@@ -49,9 +49,9 @@ class FrontendSignerTest : public ::testing::Test {
         verifier_(TestSigner::DefaultLogSigVerifier(),
                   new MerkleVerifier(new Sha256Hasher())),
         base_(make_shared<libevent::Base>()),
-        etcd_client_(base_),
-        store_(&etcd_client_, "/root", "id"),
         event_pump_(base_),
+        etcd_client_(base_),
+        store_(base_.get(), &etcd_client_, "/root", "id"),
```

Comment:
Empirically, yes :)
The exclusion about the executor in the comment only really covers a quite particular case and I guess these tests don't hit it, but maybe in the interests of not being bitten by this in the future I've changed these to use `ThreadPool`s too.